### PR TITLE
DOC-1723: Edit descriptions for networking index page

### PR DESCRIPTION
**Pull request by cmilesb**
Staging link: https://docs.redis.com/staging/jira-doc-1723/rs/networking/

### CODE REVIEW

File: content/rs/networking/_index.md [hunk 1]
```diff
@@ -16,4 +16,25 @@ When designing a Redis Enterprise Software solution, there are some
 specific networking features that are worth your time to understand and
 implement.
 
-{{< allchildren style="h2" description="true" />}}
+## [Configure cluster DNS]({{< relref "/rs/networking/cluster-dns" >}})
+
+Configure DNS to communicate between cluster nodes.
+
+- [AWS Route53 DNS management]({{< relref "/rs/networking/configuring-aws-route53-dns-redis-enterprise" >}})
+- [Client Prerequisites for mDNS]({{< relref "/rs/networking/mdns" >}}) for development and test environments
```
Comment (rrelledge) [inline content/rs/networking/_index.md:24]:
```suggestion
- [Client prerequisites for mDNS]({{< relref "/rs/networking/mdns" >}}) for development and test environments
```

File: content/rs/networking/_index.md [hunk 2]
```diff
@@ -16,4 +16,25 @@ When designing a Redis Enterprise Software solution, there are some
...
+
+Set up a load balancer to direct traffic to cluster nodes when DNS is not available.
+
+## [Multi-IP and IPv6]({{< relref "/rs/networking/multi-ip-ipv6" >}})
+
+Requirements for using multiple IP addresses or IPv6 addresses with Redis Enterprise Software.
+
+## [Network port configurations]({{< relref "/rs/networking/port-configurations" >}})
+
+Describes the various port ranges that Redis Enterprise Software uses.
```
Comment (rrelledge) [inline content/rs/networking/_index.md:36]:
```suggestion
Describes the port ranges that Redis Enterprise Software uses.
```

File: content/rs/networking/_index.md [hunk 3]
```diff
@@ -16,4 +16,25 @@ When designing a Redis Enterprise Software solution, there are some
 specific networking features that are worth your time to understand and
```
Comment (rrelledge) [inline content/rs/networking/_index.md:16]: Maybe consider reworking the intro sentence a bit.

**Review by rrelledge — COMMENTED** [review]
Looks good to me. I just left a few small suggestions.

**Review by kaitlynmichael — APPROVED** [review]

**Pull request merged**